cluster-scope/base/ocs.openshift.io/storageclusters/ocs-external-storagecluster/storagecluster.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: ocs.openshift.io/v1
+kind: StorageCluster
+metadata:
+    annotations:
+        uninstall.ocs.openshift.io/cleanup-policy: delete
+        uninstall.ocs.openshift.io/mode: graceful
+    name: ocs-external-storagecluster
+    namespace: openshift-storage
+spec:
+    externalStorage:
+        enable: true
+    version: 4.8.0
```

Comment:
How did you figure out how to create this manifest?

Reply:
I believe that finalizers are generally set up automatically, rather than in the manifest. You'll see that e.g. all namespaces have...

```
finalizers:
  - kubernetes
```

...even though we never put that in the manifests.